### PR TITLE
change unless else to if else

### PR DESCRIPTION
**Pull request by bear454**
(cherry picked from commit 7fa38e2697a8bb5da288ec5b264b7ac95612203b)

**Checklist**

- [x] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
- [x] My branch is up-to-date with the upstream `master` branch.
- [x] The tests pass locally with my changes.

**Short description of what this resolves/which [issues](https://github.com/openSUSE/osem/issues) does this fix?:**

- the condition was unnecessarily complex by using unless with an else clause.

**Changes proposed in this pull request:**

- Simplify logic by reversing behavior to an if/else.
- *Bonus*: fix a typo

### CODE REVIEW

File: spec/models/ticket_spec.rb [hunk 1]
```diff
@@ -192,7 +192,7 @@
     end
   end
 
-  describe 'currency updation' do
+  describe 'currency update' do
```
Comment (Ana06) [inline spec/models/ticket_spec.rb]: how is this related to the change? it was also not in the cherry picked commit. :confused:

Comment (bear454) [reply spec/models/ticket_spec.rb]: It was from a second commit in the parent PR; I thought it trivial; but I can submit as a separate PR.
